client/src/dolbyio_rest_apis/streaming/models/subscribe_token.py

```diff
@@ -0,0 +1,78 @@
+"""
+dolbyio_rest_apis.streaming.models.subscribe_token
+~~~~~~~~~~~~~~~
+
+This module contains the models used by the Subscribe Token models.
+"""
+
+from typing import List
+from dolbyio_rest_apis.core.helpers import get_value_or_default
+
+class SubscribeTokenStream(dict):
+    """The :class:`SubscribeTokenStream` object, which represents a Subscribe Token Stream."""
+
+    def __init__(self, dictionary: dict):
+        dict.__init__(self, dictionary)
+
+        self.stream_name = get_value_or_default(self, 'streamName', None)
+        self.is_regex = get_value_or_default(self, 'isRegex', False)
+
+class SubscribeToken(dict):
+    """The :class:`SubscribeToken` object, which represents a Subscribe Token."""
+
+    def __init__(self, dictionary: dict):
+        dict.__init__(self, dictionary)
+
+        self.id = get_value_or_default(self, 'id', None)
+        self.label = get_value_or_default(self, 'label', None)
+        self.token = get_value_or_default(self, 'token', None)
+        self.added_on = get_value_or_default(self, 'addedOn', None)
+        self.expires_on = get_value_or_default(self, 'expiresOn', None)
+        self.expires_on = get_value_or_default(self, 'expiresOn', None)
```

Comment:
@FabienLavocat copy and paste err?

Reply:
Oh yes, that line does not need to be there, I'll fix it, thanks!